docs/how-to/deploy_cups_sdcore.md

```diff
@@ -0,0 +1,137 @@
+# Deploy SD-Core with CUPS
+
+This guide covers how to install a multi-node SD-Core with Control Plane and User Plane Separation (CUPS).
+
+## Requirements
+
+- Juju 3.1 or higher has been installed
+- A load balancer type controller has been bootstrapped
+- The Juju controller is externally reachable from both Kubernetes clusters
+- Both the Kubernetes clusters have been added to the Juju controller as clouds
+
+### Specific Requirements for Control Plane Kubernetes cluster
+
+The Control Plane Kubernetes cluster has specific requirements that must be met before continuing.
+
+- A load balancer (such as MetalLB) has been configured to expose 1 IP address for the Access and Mobility Management Function (AMF)
+
+### Specific Requirements for User Plane Kubernetes cluster
+
+The User Plane Kubernetes cluster has specific requirements that must be met before continuing.
+
+- Multus has been enabled
+- MACVLAN interfaces for access and core are available to Kubernetes
+- A load balancer (such as MetalLB) has been configured to expose 1 IP address for the User Plane Function (UPF)
+
+## Guide Sample Values
+
+For the purpose of this guide, the following values will be used.
+
+### Networks
+ 
+ | Name | Subnet | Gateway IP |
+ | ---- | ------ | ---------- |
+ | access | 10.202.0.0/24 | 10.202.0.1 |
+ | core   | 10.203.0.0/24 | 10.203.0.1 |
+ | gNB    | 10.204.0.0/24 | 10.204.0.1 |
+
+
+### Domain Name Server Entries
+
+| Name | IP Address | Usage |
+| ---- | ---------- | ----- |
+| `upf.core` | 10.201.0.151 | Load balancer IP address for UPF |
+| `amf.core` | 10.201.0.201 | Load balancer IP address for AMF |
+
+### MACVLAN Interfaces
+
+| MACVLAN | Purpose |
+|---------|---------|
+| `access` | Maps to the `access` subnet |
+| `core`   | Maps to the `core` subnet |
+
+### Juju Clouds
+
+| Cloud Name | Purpose |
+|------------|---------|
+| `control-plane-cluster` | Represents the Kubernetes cluster that executes all control plane functions |
+| `user-plane-cluster`    | Represents the Kubernetes cluster that executes all user plane functions |
```

Comment:
```suggestion
```

Reply:
Then we need to remove the usage of these clouds later?  Or do we just use them without defining what they are?